src/importer.js

```diff
@@ -109,7 +109,7 @@ export const htmlToBlock = (nodeName, node) => {
       type: Block.UNSTYLED,
       data: {},
     };
-  } else if (nodeName === 'pre' && node.className && node.className === 'md-block-code-block') {
+  } else if (nodeName === 'div' && node.className && node.className === 'md-block-code-container') {
```

Comment:
This will also need the check for `pre` and old className `md-block-code-block` so that html exported with older version of `medium-draft` is compatible with newer versions.